<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBroadChatDialog</class>
 <widget class="QDialog" name="CreateBroadChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>176</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>176</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#CreateBroadChatDialog {
	background-color: rgb(245, 245, 245);
	border-radius: 20px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readyButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
       </property>
       <property name="text">
        <string>Готово</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="complaintEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: rgb(245, 245, 245);
	height: 50px;
	color: rgb(60, 47, 50);
	margin-top: 5px;
	padding-left: 10px;
	border-bottom: 3px solid rgb(60, 47, 55);
	border-radius: 1px;
	font-size: 20px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="descriptionLabel">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(65, 47, 50);
	padding-left: 10px;
	height: 30px;
}</string>
     </property>
     <property name="text">
      <string>Описание.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="privateButton">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
color: rgb(65, 47, 50);
padding-left: 10px;
height: 30px;
}

QCheckBox:checked 
{ 
color: rgb(122, 71, 50)
padding-left: 10px;
height: 30px;
}</string>
     </property>
     <property name="text">
      <string>Закрытый</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
